src/select/select.js

```diff
@@ -55,7 +55,7 @@ class Select extends React.Component<PropsT, StatelessStateT> {
     filteredOptions: null,
     textValue: this.props.textValue,
     selectedOptions: this.props.selectedOptions,
-    isDropDownOpen: this.props.type === TYPE.search,
+    isDropDownOpen: !this.props.disabled && this.props.type === TYPE.search,
```

Comment:
@blackswanny why does this default to `true` for search to begin with? i've never seen a select element or search bar that has the dropdown open by default. maybe I'm missing something.

Reply:
probably in case if we have some populated. Ok, seems like it works by different condition. I will remove this

Reply:
@gergelyke can you set this boolean to false here, and if nothing breaks, lets leave it like this

Reply:
did you wanted to mention @nudge ? I'd rather wait for him and not modify this PR

Reply:
sure. I think this condition is overriden or ignored in initial render, so it doesn't cause issues, however it is better to remove it
cc @nudge

Reply:
Removed this condition.